gpu_multi_process_run.sh

```diff
@@ -145,7 +145,7 @@ resolve_coordinator_ip
 set -e
 
 PIDS=()
-${COMMAND} &
```

Comment:
this seems right

Reply:
Yes, this makes it possible to chain commands as usual when we pass command string to XPK